src/views/OrderDetail.vue

```diff
@@ -187,37 +248,59 @@ export default defineComponent({
         console.error(error)
       }
     }
-   },
+  },
   async mounted() {
     await this.getOrderDetail(this.$route.params.orderId, this.$route.params.orderPartSeqId);
     await this.getCustomerContactDetails()
+    await this.fetchRejectReasons();
   },
-  setup () {
+  setup() {
     const store = useStore();
     const router = useRouter();
 
     return {
       Actions,
-      callOutline,
+      bagHandleOutline,
+      bagRemoveOutline,
+      copyOutline,
       copyToClipboard,
+      closeCircleOutline,
+      checkmarkCircleOutline,
       hasPermission,
-      mailOutline,
       router,
       store,
-      swapVerticalOutline,
+      timeOutline,
+      mailOutline,
       sendOutline
     };
   }
 });
 </script>
 
 <style scoped>
-main {
-  max-width: 445px;
-  margin: var(--spacer-base) auto 0; 
-}
-
 .border-top {
   border-top: 1px solid #ccc;
 }
+
+@media (min-width: 768px) {
+  main {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-areas: 'left right';
+    gap: var(--spacer-base);
+    margin: var(--spacer-lg) var(--spacer-lg) 0;
+  }
+
+  aside {
+    grid-area: right;
```

Comment:
```suggestion
    grid-column: 2;
```

Reply:
```
aside {
    grid-column: 2;
    grid-row: 1;
}
```

I also have to use the grid-row. If I don't, it places the section (order items) in the second row first col.

**For css:** 

```
aside {
   grid-column: 2;
}

```
![Screenshot from 2023-10-05 11-23-27](https://github.com/hotwax/bopis/assets/67117461/380109a5-1711-4ac4-866c-6c0d62c20041)


**For css:** 

```
aside {
    grid-column: 2;
    grid-row: 1;
}
```

![Screenshot from 2023-10-05 11-22-55](https://github.com/hotwax/bopis/assets/67117461/0749251e-8362-4011-96ba-8dcbd2e50d0c)